queries.sql

```diff
@@ -0,0 +1,93 @@
+pragma foreign_keys = on; -- sqlite only!
+
+create table organization (
+  id integer primary key autoincrement,
+  name varchar(128)
+);
+
+create table user (
+  id integer primary key autoincrement,
+  name varchar(128)
+);
+
+create table channel (
+  id integer primary key autoincrement,
+  name varchar(128),
+  organization_id integer references organization(id)
+);
+
+create table message (
+  id integer primary key autoincrement,
+  content text,
+  post_time timestamp not null default(datetime()),
+  user_id integer references user(id),
+  channel_id integer references channel(id)
+);
+
+create table user_channel (
+  user_id integer references user(id),
+  channel_id integer references channel(id)
+);
+
+insert into organization (name) values ("Lambda School");
+
+insert into user (name) values ("Alice");
+insert into user (name) values ("Bob");

```

Comment:
It is also possible to write insertion query with single line statement by separating values with a coma.